apostrophecms/apostrophe
# Pro 3805 fix rich text

[https://linear.app/apostrophecms/issue/PRO-3805/rich-text-bug](https://linear.app/apostrophecms/issue/PRO-3805/rich-text-bug)

## Summary

Fix rich text issue where it's not possible to align new text nodes.
This is because text align is configured for `heading` and `paragraph` but when we create a new node, by default it's a defaultNode, which is a paragraph too but with another name. 

By the way I upgraded every extension to the last same beta version.

## What are the specific steps to test this change?

* Go to any site rich text
* create a new node by pressing enter
* write some text and verify you can align it
* pass some text in bullet list then back to default node and it should work also.

## What kind of change does this PR introduce?


## Make sure the PR fulfills these requirements:
